pyupgrade/_plugins/native_literals.py

```diff
@@ -19,15 +19,33 @@
 SIX_NATIVE_STR = frozenset(('ensure_str', 'ensure_text', 'text_type'))
 
 
-def _fix_native_str(i: int, tokens: list[Token]) -> None:
+def _fix_native_literal(i: int, tokens: list[Token]) -> None:
```

Comment:
ah ok, yeah the confusion here -- "native" refers to that `str` is the type of `"a string"` independent of python2 / python 3

this function would probably be simpler by having a parameter which is the intended "empty" value -- rather than having to double-check and add if statements

Reply:
sorry, I don't fully understand this yet, so you mean something like:
```python
def _fix_something_something(i: int, tokens: list[Token], *, empty: str | bytes) -> None:
    ...
    else:
        tokens[i:end] = [tokens[i]._replace(name='STRING', src=empty)]
```
but how can we pass this along when yielding the `TokenFunc` at the end?

Reply:
`git grep` around for `functools.partial` -- there's a bunch of examples

it wouldn't be `str | bytes` -- just `str` (it's source)